src/Xamarin.Android.Build.Tasks/Tasks/GetAdditionalResourcesFromAssemblies.cs

```diff
@@ -203,14 +199,14 @@ bool IsValidDownload(string file, string sha1)
 			if (string.IsNullOrEmpty (sha1))
 				return true;
 
-			var hashFile = file + ".sha1";
+			var hashFile = file + ".hash";
```

Comment:
This isn't going to work. Take a look at var sha1sum = attributeValue.NamedArguments.FirstOrDefault (p => p.Name == "Sha1sum");.

the `Sha1sum` value is embedded into the attribute in the assembly. unless we alter all the old components/Nuget Package to use the new hash algorithm this is going to fail.